site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -348,6 +348,37 @@ Workloads as possible, preferring Workloads with these characteristics:
 - Workloads with the lowest priority.
 - Workloads that have been admitted more recently.
 
+## FlavorFungibility
+
+When there is not enough nominal quota of resources in a ResourceFlavor, the incoming workload can borrow or preempt in ClusterQueue or Cohort. 
+
+By default the incoming workload will stop trying the next flavor if it can get enough resource by borrowing. 
+And preemption will be triggered after all ResourceFlavors were considered.
```

Comment:
```suggestion
And Kueue triggers preemption after considering all ResourceFlavors.
```
Changing to active voice. I assume the subject of the sentence is Kueue, but feel free to change it,